### PR TITLE
Added mod info in item tooltips

### PR DESCRIPTION
**Pull request by Quent42340**
This feature will only work on the item list though.

### CODE REVIEW

File: internal.lua [hunk 1]
```diff
@@ -240,7 +240,10 @@ function unified_inventory.get_formspec(player, page)
 						..(8.2 + x * 0.7)..","
 						..(ui_peruser.formspec_y + ui_peruser.page_y + y * 0.7)..";.81,.81;"
 						..name..";item_button_"..dir.."_"
-						..unified_inventory.mangle_for_formspec(name)..";]"
+						..unified_inventory.mangle_for_formspec(name)..";]"						.."tooltip["
```
Comment (SmallJoker) [inline internal.lua]: Missing new line. If you're familiar with `string.format` I'd very welcome it if you could make use of that function to make the formspec more readable.

File: internal.lua [hunk 2]
```diff
@@ -241,6 +241,9 @@ function unified_inventory.get_formspec(player, page)
 						..(ui_peruser.formspec_y + ui_peruser.page_y + y * 0.7)..";.81,.81;"
 						..name..";item_button_"..dir.."_"
 						..unified_inventory.mangle_for_formspec(name)..";]"
+						.."tooltip[item_button_"..dir.."_"
+						..unified_inventory.mangle_for_formspec(name)..";"..item.description.."\n"
+						..minetest.get_color_escape_sequence("#2d62b7")..item.mod_origin.."]"
```
Comment (SmallJoker) [inline internal.lua]: The variable `item` is never used, thus all table contents will always be `nil`. Please test your changes.

Comment (Quent42340) [reply internal.lua]: Woops sorry, I made the changes on a previous version so I used GitHub online editor to make the PR, not the smartest idea.
Everything should be ok now, I cleaned commit log too.

**Comment by SmallJoker** [comment]
Not a good color choice.

0.4.17.1:
Blue on Green

5.0.0-dev:
Blue on Blue

What do you think about using the same format as what you get with the setting `tooltip_append_itemname`? (5.0-dev feature)
-> `Item Description [mod:name]`

**Comment by Quent42340** [comment]
I think I used the same color as the mod `witt` to make them match, because it already provides this kind of feature. Though I may agree that this color doesn't match well with the tooltip background color.
Unfortunately, I'm colorblind, so I'll let the color stuff up to you.

And I like this format, it matches `witt` too and I think it is way much clearer. Though, I planned to make the same string treatment than `witt`, which is capitalizing words and replacing `_` by spaces.

However, I like the idea of getting the full node name/alias `mod:name` so maybe we could add a toggle in the mod allowing player to display it or not. The text would need to be a bit darker than the item name to ease readability, and a new line should be added between item description and `mod:name` (which is not the case in 5.0-dev).

**Comment by Quent42340** [comment]
So besides color, what changes are still needed?

**Comment by SmallJoker** [comment]
None, I think. This PR isn't on my priority list (actually I just forgot about it).

**Comment by SmallJoker** [comment]
Added as 54abad7. Thanks for your contribution.

**Pull request closed without merging**